### PR TITLE
feat: create cli

### PR DESCRIPTION
**Pull request by lee-tammy**
The initial implementation of the cli -- anything I should add/change?

### CODE REVIEW

File: src/cli.ts [hunk 1]
```diff
@@ -1,7 +1,66 @@
...
+ 
+    Options
+      --help        Prints this help message.
+  `,
+  flags: {
+    help: {type: 'boolean'}
+  }
+});
+
+if(cli.input.length !== 1){
```
Comment (jplaisted) [inline src/cli.ts]: Some formatting issues in this file. Can you run clang format?

Comment (jplaisted) [reply src/cli.ts]: Specifically needs to be a space after the `if`, and before the `{`. Then below around the `else`.

File: src/cli.ts [hunk 2]
```diff
@@ -1,7 +1,66 @@
...
+  console.error(`Error: should have 1 argument, but got ${cli.input.length}`)
+  cli.showHelp(1);
+  process.exit(1);
+}else{
+  run(cli.input[0]);
+}
+
+async function validNodePackage(path: string){
+  try{
+    await pify(fs.stat)(path)
```
Comment (jplaisted) [inline src/cli.ts]: Missing semicolon

Comment (jplaisted) [reply src/cli.ts]: (They're optional in JavaScript but our coding style / standards enforce them)

File: src/cli.ts [hunk 3]
```diff
@@ -1,7 +1,66 @@
...
+    console.error(`Error: directory does not contain a package.json file`)
+    cli.showHelp(1);
+    return false;
+  }
+  return true;
+}
+
+async function run(packageRootDir: string) {
   // Step 1: Takes in the root of the package
+  if(!validNodePackage(packageRootDir)){
```
Comment (jplaisted) [inline src/cli.ts]: `validNodePackage()` returns a promise for a boolean, not a boolean. You need to `await`.

File: src/cli.ts (hunk 1 above)
Comment (lee-tammy) [reply src/cli.ts]: done

File: src/cli.ts (hunk 3 above)
Comment (lee-tammy) [reply src/cli.ts]: done

File: src/cli.ts (hunk 2 above)
Comment (lee-tammy) [reply src/cli.ts]: done

File: src/cli.ts [hunk 4]
```diff
@@ -1,7 +1,64 @@
...
+  process.exit(1);
+} else {
+  run(cli.input[0]);
+}
+
+async function validNodePackage(path: string) {
+  try {
+    await pify(fs.stat)(path);
+  } catch (err) {
+    console.error(`${err}`);
```
Comment (jplaisted) [inline src/cli.ts]: Just log the error, no need to wrap in a template literal.

File: package.json [hunk 5]
```diff
@@ -5,14 +5,16 @@
   "main": "index.js",
   "dependencies": {
     "acorn": "^5.7.1",
-    "node": "^10.6.0",
+    "meow": "^5.0.0",
+    "node": "^10.8.0",
```
Comment (kjin) [inline package.json]: While we're here -- let's remove this dependency, I think it's unneeded.

File: src/cli.ts (hunk 4 above)
Comment (lee-tammy) [reply src/cli.ts]: done

File: package.json (hunk 5 above)
Comment (lee-tammy) [reply package.json]: done

**Pull request merged**